examples/consul/README.md

```diff
@@ -0,0 +1,19 @@
+# Consul
+
+This example holds a simple Semaphore HTTP implementation with a service reslved by Consul service discovery
+
+## Getting started
+
+To run this example you need to have Go 1.13> installed on your machine.
+
+Prepare the environment:
+
+1. Run consul `consul agent -dev --config-dir=./awesome-dogs/consul.d -ui`
```

Comment:
let's use a docker container instead.

Reply:
I suppose people who use consul know how to run consul. I will better change the doc a bit